test/components/views/right_panel/__snapshots__/RoomHeaderButtons-test.tsx.snap

```diff
@@ -0,0 +1,35 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`RoomHeaderButtons-test.tsx should render 1`] = `
+<DocumentFragment>
+  <div
+    aria-current="false"
+    aria-label="Chat"
+    class="mx_AccessibleButton mx_RightPanel_headerButton mx_RightPanel_timelineCardButton"
+    role="button"
+    tabindex="0"
+  />
+  <div
+    aria-current="false"
+    aria-label="Threads"
+    class="mx_AccessibleButton mx_RightPanel_headerButton mx_RightPanel_headerButton_unread mx_RightPanel_threadsButton"
```

Comment:
`mx_RightPanel_headerButton_unread` is a bug here (see: https://github.com/vector-im/element-web/issues/25284)

Reply:
I think you need to pull develop and update this snapshot